module/Company/view/company/admin/edit-company.phtml

```diff
@@ -71,64 +71,63 @@ $submit->setAttribute('class', 'btn btn-primary pull-right');
 <h2><?= $this->translate('Packages') ?></h2>
 <table class="table table-striped">
     <thead>
-        <tr>
-            <th>
-                <?= $this->translate('Active'); ?>
-            </th>
-            <th>
-                <?= $this->translate('Start date'); ?>
-            </th>
-            <th>
-                <?= $this->translate('Expiration date'); ?>
-            </th>
-            <th>
-                <?= $this->translate('Jobs / Type'); ?>
-            </th>
-            <th>
-                <?= $this->translate('Actions'); ?>
-            </th>
-            </th>
-        </tr>
+    <tr>
+        <th>
+            <?= $this->translate('Active'); ?>
+        </th>
+        <th>
+            <?= $this->translate('Start date'); ?>
+        </th>
+        <th>
+            <?= $this->translate('Expiration date'); ?>
+        </th>
+        <th>
+            <?= $this->translate('Jobs / Type'); ?>
+        </th>
+        <th>
+            <?= $this->translate('Actions'); ?>
+        </th>
+    </tr>
```

Comment:
I don't see why we would have to remove one level of indentation here.

Reply:
I agree. PhpStorm thought this was neccesarry to do somehow. But looking at the opening and closing tags it should really have one more level.